sdk/servicebus/azservicebus/eng/azure-pipelines.yml

```diff
@@ -0,0 +1,95 @@
+variables:
```

Comment:
I would recommend copy and pasting a ci.yml file from azcore

Reply:
Agreed - I'll need to work that out with @benbp

Reply:
Tracking here: #15245